force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -228,18 +281,24 @@ public with sharing class ClassesObjectsMethods {
     public class Book {
         // Declare three public instance variables - title, author, pages
         // title
+        public String title;
         // author
+        public String author;
         // pages
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String initialTitle, String initialAuthor) {
             // Initialize the title and author
+            this.title = initialTitle;
+            this.author = initialAuthor;
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            return 'Title: ' + title + ', Author: ' + author; // Replace null with the variable you used to store the result
```

Comment:
```suggestion
            return 'Title: ' + this.title + ', Author: ' + this.author; // Replace null with the variable you used to store the result
```
Even though it's not required in apex, try to use the this keyword for best practice